index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" media="screen" href="css/style.css">
+
```

Comment:
Please avoid leaving empty code lines inside the tags, this will show up better in your code, and it avoids thoughts about lack of relevant information.

Reply:
- [ ] [OPTIONAL] Good job,  you sounded polite with this comment and the change requested was also well explained :100: . you can try using a couple of emojis and/or gifs in the review to make the code review more human-friendly. :smile_cat: